portafly/src/components/pages/accounts/listing/ActionButtonImpersonate.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Button } from '@patternfly/react-core'
+import { BoltIcon } from '@patternfly/react-icons'
+
+const ActionButtonImpersonate = () => {
+  const { t } = useTranslation('audienceAccountsListing')
+
+  const onClick = () => {
+    // TODO
```

Comment:
Todo what?

Reply:
Todo the onClick for the "Impersonate" action.